sdk/digitaltwins/digitaltwins/review/digitaltwins.api.md

```diff
@@ -0,0 +1,360 @@
+## API Report File for "@azure/digitaltwins"
+
+> Do not edit this file. It is a report generated by [API Extractor](https://api-extractor.com/).
+
+```ts
+
+import * as coreHttp from '@azure/core-http';
+import { OperationOptions } from '@azure/core-http';
+import { PagedAsyncIterableIterator } from '@azure/core-paging';
+import { PageSettings } from '@azure/core-paging';
+import { RequestOptionsBase } from '@azure/core-http';
+import { RestResponse } from '@azure/core-http';
+import { TokenCredential } from '@azure/core-http';
+
+// @public
+export interface AzureDigitalTwinsAPIOptionalParams extends coreHttp.ServiceClientOptions {
+    $host?: string;
```

Comment:
Let's find a better name for this.

Reply:
Added to suggested swagger changes

---
In reply to: [473240787](https://github.com/Azure/azure-sdk-for-js/pull/10560#discussion_r473240787) [](ancestors = 473240787)